### PR TITLE
[FIX] Don't reset is_admin flag when account is updated

### PR DESCRIPTION
**Pull request by MatthewPhinney**

### CODE REVIEW

**Comment by MatthewPhinney** [comment]
We were defaulting the `is_disabled` and `is_admin` flags to `false` in the code. This means that if a user updated an account with one of those flags already set to `true`, but did not explicitly specify the flag in their JSON request, it would be reset to `false`.

File: src/models/db/accounts.js [hunk 1]
```diff
@@ -43,8 +41,12 @@ function convertToPersistent (data) {
       data.minimum_allowed_balance = Number(data.minimum_allowed_balance)
     }
   } // Otherwise the db defaults minimum_allowed_balance to 0
```
Comment (lumberj) [inline src/models/db/accounts.js]: Is this comment relevant to the code below it?

Comment (MatthewPhinney) [inline src/models/db/accounts.js]: No. I added that because @gip wanted a comment for #272 on how the minimum balance is handled in the db.

**Comment by lumberj** [comment]
LGTM
